docs/modules/SDDs/pages/0028-reusable-config-packages.adoc

```diff
@@ -45,33 +45,49 @@ This gives us change history and versioning of configuration packages based on a
 Making each configuration package a Git repository also fits well with the Project Syn approach of keeping all dependencies (global and tenant config, components) in Git.
 We can also profit from all the already existing logic for cutting releases, and CI tests which we've engineered for Commodore components.
 
+=== Specifying configuration packages
+
+We propose a new top-level parameter `configuration_packages`.
```

Comment:
Why not just `packages`?

Reply:
I've no hard feelings either way, I'd be fine with using `packages`.

Reply:
Changed to `packages` in the latest version